.github/workflows/ci.yml

```diff
@@ -261,28 +246,23 @@ jobs:
 
       - name: Ensure cache is healthy
         if: ${{ matrix.os != 'macos-latest' && steps.cache-pip.outputs.cache-hit == 'true' }}
-        run: timeout 10s poetry run pip --version || rm -rf .venv
+        run: timeout 10s pip --version || rm -rf .venv
 
       - name: Ensure cache is healthy (MacOS)
         if: ${{ matrix.os == 'macos-latest' && steps.cache-pip.outputs.cache-hit == 'true' }}
         run: |
           brew install coreutils
-          timeout 10s poetry run pip --version || rm -rf .venv
+          timeout 10s pip --version || rm -rf .venv
 
       - name: Install dependencies
         if: steps.cache-pip.outputs.cache-hit != 'true' || needs.diff.outputs.related == 'true'
-        run: poetry install -E "model-server docs"
+        run: pip install "." pytest pytest-asyncio pandas
 
-      - name: Run tests and generate coverage report
-        run: make tests-general_features
+      - name: Change to project directory
+        run: cd ./tests/e2e/bento_server_general_features
 
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v2
-        with:
```

Comment:
Can you restore this since we are using carryforward flags for codecov to transfer previous reports on main branch